File: 3581-the-two-sneaky-numbers-of-digitville/the-two-sneaky-numbers-of-digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) 
    {
        vector<int> ans;
        unordered_set<int> set;
        for(int num : nums)
        {
            if(set.count(num)>0)
            ans.push_back(num);

            set.insert(num);
        }
        return ans;
    }
};

File: 3581-the-two-sneaky-numbers-of-digitville/the-two-sneaky-numbers-of-digitville.java
class Solution {
    public int[] getSneakyNumbers(int[] nums) {
        Set<Integer> set = new HashSet<>();
        int index = 0;
        int[] res = new int[2];
        for(int num : nums){
            if(set.contains(num)){
                res[index++] = num;
            }
            set.add(num);
        }
        return res;
    }
}